src/AutoBuild.jl

```diff
@@ -343,7 +343,7 @@ function build_tarballs(ARGS, src_name, src_version, sources, script,
         # The location the binaries will be available from
         bin_path = "https://github.com/$(deploy_jll_repo)/releases/download/$(tag)"
         build_jll_package(src_name, build_version, sources, code_dir, build_output_meta,
-                          dependencies, bin_path; verbose=verbose, extra_kwargs...)
+                          dependencies, bin_path; verbose=verbose, julia_compat=julia_compat, extra_kwargs...)
```

Comment:
Let's use the shorter form
```suggestion
                          dependencies, bin_path; verbose, julia_compat, extra_kwargs...)
```